extensions/SiteMapIndex/template/en/default/hook/global/robots-end.txt.tmpl

```diff
@@ -0,0 +1,20 @@
+User-agent: *
```

Comment:
Rename this file to xtensions/SiteMapIndex/template/en/default/hook/robots-end.txt.tmpl